docs/welcome/key-metrics-guide.md

```diff
@@ -0,0 +1,69 @@
+---
+id: key-metrics-guide
+title: "Key Metrics Guide"
+sidebar_label: "Key Metrics Guide"
+keywords:
+  - "metrics"
+  - "openSauced"
+  - "OpenSauced metrics guide"
+  - "Key metrics guide"
+  - "Open Source metric guide"
+---
+
+Welcome to the Key Metrics Guide, where we define the main metrics used on our platform to help you better understand them.
+
+## OSCR
+
+[OSCR](../glossary/#oscr) stands for Open Source Contributor Rating. It's a way to measure and rate the impact of your open-source contributions over 90 days.
+
+This helps you understand the quality and frequency of your contributions. It's a good way to measure your open source activities and takes into account many factors.
```

Comment:
Let's change this to third person and add a little more context:

This metric helps to determine the quality and frequency of a contributor's contributions and is determined by their ability to engage in conversations about their work and the project, their rate of contributions to the projects they engage with, and the quality of their contributions.